### PR TITLE
get-started: link to external command line how tos

### PR DESCRIPTION
**Pull request by jost-s**

### CODE REVIEW

File: src/pages/get-started/index.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ Holochain is supported under WSL2 via the Ubuntu distribution.
 
 Once you've ensured that your system meets the hardware requirements and set up WSL2 on Windows or a dual boot Linux OS (if applicable), you can proceed with the installation of the Nix package manager and the binary package cache for Holochain.
 
-Open a [command line terminal](https://hackmd.io/c15fobj9QtmOuEuiNAkaQA) and run the following command by pasting or typing the following text in and pressting <kbd>Enter</kbd>:
+Open a command line terminal ([on Linux](https://ubuntu.com/tutorials/command-line-for-beginners#3-opening-a-terminal), [on macOS](https://support.apple.com/en-gb/guide/terminal/pht23b129fed/mac)) and run the following command by pasting or typing the following text in and pressting <kbd>Enter</kbd>:
```
Comment (pdaoust) [inline src/pages/get-started/index.md:84]: How different is WSL2 from the Ubuntu instructions? Not having used WSL in forever, I don't know if you're working in the GNOME GUI or if you open a special WSL terminal app. If the latter, I wonder if MS has some instructions out there we can link to.

**Review by pdaoust — COMMENTED** [review]
Fantastic; I'd searched for some more permanent third party tutorials and didn't end up finding any. (NB: I've checked the "how to open a terminal" links and they're great; didn't check the IDEs' documentation on how to add a command to $PATH)

File: src/pages/get-started/index.md (hunk 1 above)
Comment (jost-s) [reply src/pages/get-started/index.md:84]: We're only supporting Ubuntu installation under WSL, so I figured this is safe. I presume that if a user is able to customize the Linux distribution under WSL, they're comfortable opening a terminal too.

File: src/pages/get-started/index.md [hunk 2]
```diff
@@ -1052,7 +1052,7 @@ CommentsForPost.svelte  EditPost.svelte
...
 
-If you don't yet have path commands for opening files in your prefered IDE, [this tutorial can help guide you through setting up path commands](https://hackmd.io/@oitz5O-qR2qrfRre3Kbv-Q/r1Z_Z6Qgrn). Going forward in this tutorial, we are going to use `code` to open files in [VS Code](https://code.visualstudio.com/), but you should substitute a different command (ex: `atom` or `vim`) for `code` if you are using a different editor.
+If you don't yet have path commands for opening files in your prefered IDE, there are instructions for [VSCode/VSCodium](https://code.visualstudio.com/docs/setup/mac#_launching-from-the-command-line), [Sublime Text](https://www.sublimetext.com/docs/command_line.html#setup) and [WebStorm](https://www.jetbrains.com/help/webstorm/working-with-the-ide-features-from-command-line.html#5d6e8844). Going forward in this tutorial, we are going to use `code` to open files in [VS Code](https://code.visualstudio.com/), but you should substitute a different command (ex: `subl` or `vim`) for `code` if you are using a different editor.
```
Comment (zippy) [inline src/pages/get-started/index.md:1055]:
```suggestion
If you don't yet have path commands for opening files in your prefered IDE, there are instructions for [VSCode/VSCodium](https://code.visualstudio.com/docs/setup/mac#_launching-from-the-command-line), [Sublime Text](https://www.sublimetext.com/docs/command_line.html#setup) and [WebStorm](https://www.jetbrains.com/help/webstorm/working-with-the-ide-features-from-command-line.html#5d6e8844). Going forward in this tutorial, we are going to use the `code` command when we mean for you to open files in your IDE, but you should substitute a different command (ex: `subl`, `vim`, `emacs` etc) for `code` if you are using a different editor.
```

**Review by zippy — APPROVED** [review]
Looks great, small text suggestion, take or leave!

File: src/pages/get-started/index.md (hunk 1 above)
Comment (jost-s) [reply src/pages/get-started/index.md:84]: It's virtually Linux - in all ways.

**Pull request merged**